params.go

```diff
@@ -23,6 +24,11 @@ type Params struct {
 	// Please use StripeAccount instead.
 	Account string `form:"-"` // Passed as header
 
+	// Context used for request. It may carry deadlines, cancelation signals,
+	// and other request-scoped values across API boundaries and between
+	// processes.
+	Context context.Context `form:"-"`
```

Comment:
I know we aren't dealing with newbie programmers, but maybe explicitly stating something like:
```
// A cancelled or timed out context does not provide any type guarantee 
// whether the operation was or was not completed on the backend.
// You must still perform appropriate rollback or retry with the same idempotency key.
```

Reply:
👍

Reply:
> provide any type guarantee 

should either read `provide any guarantee` or `provide any type of guarantee`

Reply:
I like it. I modified your comment a bit and committed in 1127516.